modules/mayaUSD.mod.template

```diff
@@ -7,7 +7,7 @@ resources:
 PYTHONPATH+:=lib/python
 USD_LOCATION:=
 MAYA_USD_VERSION=${USD_VERSION}
-PXR_MTLX_STDLIB_SEARCH_PATHS+=${USD_INSTALL_LOCATION}/libraries
+PXR_MTLX_STDLIB_SEARCH_PATHS+:=libraries
```

Comment:
This sets the variable relative to the folder defined for the "USD" module (line 1). So for Maya 2022 it will have the correct 2/3 in the path (for python version).